tests/integration/transfer-fees/transfer-fees.spec.ts

```diff
@@ -0,0 +1,281 @@
+    });
+
+    it('validates that custom fee has more than 6 decimal places', async () => {
+      await sendForm.inputToAmountField('0.000001');
+      await sendForm.inputToAddressField('ST3TZVW4VTZA1WZN2TB6RQ5J8RACHZYMWMM2N1HT2');
+      await sendForm.clickFeeEstimateSelect('$standardFeeSelect');
+      await sendForm.selectCustomFeeOption();
+      await sendForm.inputFee('0.0000001');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$feeError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toEqual('STX can only have 6 decimals');
+    });
+
+    it('validates that custom fee must be positive', async () => {
+      await sendForm.inputToAmountField('0.000001');
+      await sendForm.inputToAddressField('ST3TZVW4VTZA1WZN2TB6RQ5J8RACHZYMWMM2N1HT2');
+      await sendForm.clickFeeEstimateSelect('$standardFeeSelect');
+      await sendForm.selectCustomFeeOption();
+      await sendForm.inputFee('-1');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$feeError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toEqual('Amount must be positive');
+    });
+
+    it('validates that the amount must be more than zero', async () => {
+      await sendForm.inputToAmountField('0');
+      await sendForm.inputToAddressField('ST3TZVW4VTZA1WZN2TB6RQ5J8RACHZYMWMM2N1HT2');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$amountFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toContain(SendFormErrorMessages.MustNotBeZero);
+    });
+
+    it('validates that the memo must be less than 34-bytes', async () => {
+      await sendForm.inputToAmountField('0.000001');
+      await sendForm.inputToAddressField('ST3TZVW4VTZA1WZN2TB6RQ5J8RACHZYMWMM2N1HT2');
+      await sendForm.inputToMemoField(
+        'lorem-ipsum-lorem-ipsum-lorem-ipsum-lorem-ipsum-lorem-ipsum-lorem-ipsum'
+      );
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$memoFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toContain(SendFormErrorMessages.MemoExceedsLimit);
+    });
+
+    it('validates that the amount field can have only 9 decimals', async () => {
+      await sendForm.inputToAmountField('0.00000000001');
+      await sendForm.inputToAddressField('ST3TZVW4VTZA1WZN2TB6RQ5J8RACHZYMWMM2N1HT2');
+      await sendForm.clickPreviewTxBtn();
+      const errorMsgElement = await sendForm.page.$$(sendForm.getSelector('$amountFieldError'));
+      const errorMessage = await errorMsgElement[0].innerText();
+      expect(errorMessage).toContain(SendFormErrorMessages.Only9Decimals);
```

Comment:
```suggestion
      expect(errorMessage).toContain('CAT can only have 9 decimals');
```
As this is a dynamic error (the symbol changes based on the token you have selected), we can  just add the error directly, rather than changing the error enum.